compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -225,3 +225,34 @@ hir_analysis_functions_names_duplicated = functions names are duplicated
 
 hir_analysis_simd_ffi_highly_experimental = use of SIMD type{$snip} in FFI is highly experimental and may result in invalid code
     .help = add `#![feature(simd_ffi)]` to the crate attributes to enable
+
+hir_analysis_impl_not_marked_default = `{$ident}` specializes an item from a parent `impl`, but that item is not marked `default`
+    .label = cannot specialize default item `{$ident}`
+    .ok_label = parent `impl` is here
+    .note = to specialize, `{$ident}` in the parent `impl` must be marked `default`
+    .err_note = parent implementation is in crate `{$cname}`
+
+hir_analysis_missing_trait_item = not all trait items implemented, missing: `{$missing_items_msg}`
```

Comment:
I wonder if there's a better way of doing this concatenation...

`$missing_items_msg` looks like

```
a`, `b`, `c
```

So maybe we should just append the first and last backtick instead of splitting it between the ftl file and the rust source.

Reply:
I think it's okay as it, or do you have a better way?